_targets.R

```diff
@@ -115,35 +117,128 @@ targets_list <- list(
     ),
     # TODO: might need to split across variables, too. Once we scale up, our queries
     # might be too large and chunking by variable could help.
-    pattern = cross(query_cells_centroids_list_by_tile, gcm_names),
-    format = "file"
+    pattern = cross(query_cells_centroids_list_by_tile, gcm_names, gcm_dates_df),
+    format = "file",
+    error = "continue"
   ),
 
   ##### Munge GDP output into NetCDF files that will feed into GLM #####
-  # TODO - munge data
+
+  # Need to munge GCM variables into useable GLM variables
+  tar_target(glm_vars_info,
+             tibble(
+               # TODO: MISSING LONGWAVE
+               var_name = c("Rain", "AirTemp", "Snow", "RelHum", "Shortwave", "Longwave", "WindSpeed"),
+               longname = c("Total daily rainfall",
+                            "Total daily snowfall derived from precipitation and air temp",
+                            "Average daily near surface air temperature",
+                            "Average daily percent relative humidity",
+                            "Average daily surface downward shortwave flux in air",
+                            "LONGWAVE EXPLANATION",
+                            "Average daily windspeed derived from anemometric zonal and anenometric meridional wind components"),
+               units = c("meters", "meters", "degrees Celcius", "", "???", "???", "???"),
+               precision = "float"
+             )),
+  tar_target(
+    gcm_data_daily_feather,
+    munge_to_glm(gcm_data_raw_feather),
+    pattern = map(gcm_data_raw_feather),
+    format = "file"
+  ),
+
+  # Create feather files that can be used in the GLM pipeline without
+  # having to be munged and extracted via NetCDF. Temporary solution
+  # while we work out some NetCDF challenges.
+  # TODO: delete once we finish the NetCDF DSG build (see issue #252)
+  tar_target(
+    out_skipnc_feather, {
+      out_dir <- "7_drivers_munge/out_skipnc"
+      purrr::map(gcm_data_daily_feather, function(fn, gcm_names, gcm_dates_df) {
+        gcm_name <- str_extract(fn, paste(gcm_names, collapse="|"))
+        gcm_time_period <- str_extract(fn, paste(gcm_dates_df$projection_period, collapse="|"))
+
+        read_feather(fn) %>%
+          split(.$cell) %>%
+          purrr::map(., function(data) {
+            cell <- unique(data$cell)
+            data_to_save <- data %>% select(-cell)
+            out_file <- sprintf("%s/GCM_%s_%s_%s.feather", out_dir, gcm_name, gcm_time_period, cell)
+            write_feather(data_to_save, out_file)
+            return(out_file)
+          })
+
+      }, gcm_names, gcm_dates_df)
+      return(out_dir)
+    },
+    format = "file"
+  ),
+
+  # Group daily feather files by GCM to map over and include
+  # branch file hashes to trigger rebuilds for groups as needed
+  tar_target(gcm_data_daily_feather_group_by_gcm,
+              build_branch_file_hash_table(names(gcm_data_daily_feather)) %>%
```

Comment:
This is a nice solution for tracking file dependencies while getting the grouping we'd like.